src/core/lib/security/credentials/tls/grpc_tls_credentials_options.h

```diff
@@ -77,6 +80,7 @@ struct grpc_tls_credential_reload_config
   }
 
  private:
+  void* context_;
```

Comment:
Thanks for making the changes. Could you please add some comments about "context" e.g., why we need it? i.e., it will be mainly used as a pointer to the wrapped language implementation of `grpc_tls_credential_reload_config`, and there is a need to access it in C-core implementation of `schedule`, `cancel`...

Reply:
Added comments to the context field in the grpc_tls_credential_reload_config and grpc_tls_server_authorization_check_config definitions.